doc/overview.rst

```diff
@@ -84,6 +84,58 @@ For more `information <https://boto3.amazonaws.com/v1/documentation/api/latest/r
     # Deletes the SageMaker model
     mxnet_predictor.delete_model()
 
+Git Support
+~~~~~~~~~~~
+The SageMaker Python SDK has Git support. If you have your training scripts in your GitHub repository, you can
+use them directly wihtout the trouble to download them to local machine. Git support can be enabled simply by
+providing ``git_config`` parameter when initializing an estimator. If git support is enabled, then ``entry_point``,
+``source_dir`` and  ``dependencies`` should all be relative paths in the Git repo. Git support works for frameworks
+images (TensorFlow, MXNet, Chainer, PyTorch, and Scikit-Learn). Note that if you decided to
+use Git support, then everything you need for ``entry_point``, ``source_dir`` and ``dependencies`` should be in
+a single Git repo.
+
+Here is an example:
+
+.. code:: python
+
+        import sagemaker
+
+        # Specifies the git_config parameter
+        git_config = {'repo': 'https://github.com/GaryTu1020/python-sdk-testing.git',
+                      'branch': 'branch1',
+                      'commit': '4893e528afa4a790331e1b5286954f073b0f14a2'}
+
+        # Configures a PyTorch Estimator (no training happens here)
+        pytorch_estimator = PyTorch(entry_point='mnist.py',
+                                    role='SageMakerRole',
+                                    source_dir='pytorch',
+                                    git_config=git_config,
+                                    framework_version=PyTorch.LATEST_VERSION,
+                                    py_version=PYTHON_VERSION,
+                                    train_instance_count=1,
+                                    train_instance_type='ml.c4.xlarge',
+                                    sagemaker_session=sagemaker_local_session)
```

Comment:
I personally prefer showing a small example only focused on git support than showing this big example here.